app/views/clients/view_client.html.erb

```diff
@@ -0,0 +1 @@
+<%= react_component('ClientTasks', {clientId: 12}) %>
```

Comment:
could you change this to be dynamic based on passed in props?

Reply:
Did I fix this correctly?

Reply:
LOL holy shit that was fast. You'll have to change the routes file: for example Amy has 
get `'/clients/:client_id/comments', to: 'clients#client_comments'`
You'll need something like get `'/client/:client_id', to: 'clients#view'`, and then make the view method in clients_controller to have something like 

`if params[:client_id]`  
      `@client = Client.find(params[:client_id])`

and then @client will be available to `view_client.html.erb`. I'd recommend passing in the whole @client with `{client: @client}` and then you can use the client_id in your component.

Reply:
How should the call to `render` look, once I have the `@client` variable?

Reply:
Alright, it's done.

Reply:
So as long as you name the ...html.erb the same as the method that's being called, you don't actually need a render. You need a render in your case because you're calling `view_clients.html.erb` from a method called `view` in `clients`. If there's no render in the controller, Rails will look for a view called `x.html.erb` in the `y` folder where `y` is the name of your controller and `x` is the name of the method you called in the controller. So if you rename `view_clients.html.erb` in the `clients` folder to `view.html.erb` rails will be smart enough to automatically render it with the provided variables (@client)

Reply:
You can Google implicit rendering if u wanna learn more about it it's pretty cool lol

Reply:
Should I make the change, or is it fine as is?